qstnrs/questionnaire/forms.py

```diff
@@ -0,0 +1,16 @@
+from django import forms
+
+
+class PageForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        page = kwargs.pop('page')
+        super(PageForm, self).__init__(*args, **kwargs)
+
+        for question in page.question_set.all():
```

Comment:
I think you can use values_list here as well: page.question_set.values_list('id', 'question_text') and then you can fetch the answers Answer.objects.values_list('id', 'answer_text'). Be aware of the fact that this will generate a query for each question. If however your db has many question, you might get performance issues. In this case you might consider iterating through something like this: page.question_set.values('id','question_text', 'answer', 'answer__answer_text')